core/src/main/java/org/apache/accumulo/core/metadata/schema/CompactionMetadata.java

```diff
@@ -111,7 +112,7 @@ public String toJson() {
     jData.groupId = cgid.toString();
     jData.priority = priority;
     jData.propDels = propagateDeletes;
-    jData.fateTxId = fateTxId;
+    jData.fateId = fateId;
```

Comment:
The following would probably make mch more readable json and be tolerant of changes to the implementation of FateId like variable renames.

```suggestion
    jData.fateId = fateId.canonical();
```